packages/console/src/pages/EnterpriseSsoDetails/index.tsx

```diff
@@ -109,13 +110,13 @@ function EnterpriseSsoConnectorDetails<T extends SsoProviderName>() {
     <DetailsPage
       backLink={enterpriseSsoPathname}
       backLinkTitle="enterprise_sso_details.back_to_sso_connectors"
-      isLoading={isLoading}
       error={requestError}
       onRetry={() => {
         void mutate();
       }}
     >
       <PageMeta titleKey="enterprise_sso_details.page_title" />
+      {isLoading && <Skeleton />}
```

Comment:
Need to use `!isLoading && ssoConnector` below, since some times the `ssoConnector` is available, but the assert services is still loading.